packages/backend/bindings/capacitor/android/src/main/java/org/iota/walletactorsystem/WalletPlugin.java

```diff
@@ -31,13 +32,12 @@ public void initialize(final PluginCall call) {
         call.setKeepAlive(true);
 
         final ActorCallback callback = response -> {
-            System.out.println("walletEvent " + response);
             JSObject walletResponse = new JSObject();
             walletResponse.put("walletResponse", response);
             notifyListeners("walletEvent", walletResponse);
         };
 
-        Actor.iotaInitialize(callback, actorId, "./database");
+        Actor.iotaInitialize(callback, actorId, "data/data/com.iota.wallet/cache/database");
```

Comment:
Can we use an API to get the path for our app instead? For example [`getCacheDir`](https://developer.android.com/reference/android/content/Context#getCacheDir())?

Reply:
yes you right, thanks! I've end up using [getFilesDir](https://developer.android.com/reference/android/content/Context#getFilesDir()) to that avoid cache evictions

Reply:
Ah good call, I was wondering about the cache evictions